mantidproject/mantid
# fill algorithm details dialog with running algs

Populates the workbench algorithm details dialog with all currently running algorithms, rather than waiting for thier first progress report.  This is particularly important for those algorithms with poor progress reporting.



**To test:**
1. Start workbench (mplot does this differently)
1. Start a few algorithms, I suggest Pause a few times with long intervals
1. Also least one of FakeISISHistoDAE or Fake...EventDAE as these do not send progress until someone connects to them.
1. Any other long running algs you can think of
1. While they are running click the Details button in the Algorithms toolbox
1. You should get an entry for all of the algorithms that are running, with a cancel button for each.
1. Those that are getting progress messages should update.
1. the cancel buttons should work (at least as well as they did before).

Re #27077



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
